### DISCUSSION TITLE
7.28活动BD图没反应

### DISCUSSION THREAD

**Discussion by BanksRao** [LmeSzinc/AzurLaneAutoScript#1449] [Q&A] [answered]
<img width="833" alt="屏幕截图 2022-07-29 144934" src="https://user-images.githubusercontent.com/69216744/181701450-64e90a30-96a8-4643-a485-8c78c0a9fd8b.png">
<img width="515" alt="屏幕截图 2022-07-29 144958" src="https://user-images.githubusercontent.com/69216744/181701470-f6a96b9c-9f85-40f6-9cf3-3ad71fd0f7e4.png">
<img width="515" alt="屏幕截图 2022-07-29 145020" src="https://user-images.githubusercontent.com/69216744/181701471-7644b8bb-b32d-4c3a-9929-3c5acd41cae0.png">
mei'fan
<img width="518" alt="屏幕截图 2022-07-29 145336" src="https://user-images.githubusercontent.com/69216744/181701475-34ef393d-2bbe-4770-adb2-da8d7d65965c.png">
<img width="524" alt="屏幕截图 2022-07-29 145402" src="https://user-images.githubusercontent.com/69216744/181701481-425312ce-75f6-4cce-a3ab-9c28516b4fb6.png">

**Comment by bookbug666** [comment, marked as answer]
#1445

**Reply by BanksRao** [reply]
=have to wait for update?

**Reply by bookbug666** [reply]
Yes, be patient. Check for updates umm few hours later?

**Reply by BanksRao** [reply]
ohhhhkay  thanks
